src/Video/RenderProgram/StaticRenderProgram.hpp

```diff
@@ -9,59 +9,77 @@ namespace Video {
     namespace Geometry {
         class Geometry3D;
     }
-    
+
     /// Render program to render an entity using default shader program.
     class StaticRenderProgram {
         public:
-            /// Create new default render program.
-            StaticRenderProgram();
-            
-            /// Destructor.
-            ~StaticRenderProgram();
+        /// Create new default render program.
+        StaticRenderProgram();
+
+        /// Destructor.
+        ~StaticRenderProgram();
+        /// Bind shadow render program.
+        /**
+            * @param viewMatrix The camera's view matrix.
```

Comment:
Missing parameter documentation. Also misaligned asterisks.

Reply:
Fixed

Reply:
`shadowId`, `shadowWidth`, `shadowHeight`, and `depthFbo` are not documented.

Reply:
Fixed